features/order/managing_orders/modifying_shipping_address/modifying_shipping_address_validation.feature

```diff
@@ -16,7 +16,7 @@ Feature: Modifying a customer's shipping address validation
         And the customer chose "Free" shipping method with "Cash on Delivery" payment
         And I am logged in as an administrator
 
-    @ui
+    @ui @no-api
```

Comment:
Why did you mark this scenarios as no-api?

Reply:
For the first time, I also considered this scenario as `@no-api`, but Grześ is right, it should not be. It should be just an empty `PUT` operation.